reana_workflow_controller/rest.py

```diff
@@ -329,6 +330,28 @@ def run_yadage_workflow_from_remote_endpoint():  # noqa
         abort(400)
 
 
+@restapi_blueprint.route('/seed', methods=['POST'])
+def seed():  # noqa
```

Comment:
workflow seeding is already present in master (you can see here the [docs](http://reana-workflow-controller.readthedocs.io/en/latest/restapi.html#post--api-workflows-workflow_id-workspace))